webrtc.html

```diff
@@ -2794,6 +2825,99 @@
+
+          <dt>double resolutionScale</dt>
+          <dd>
+            <p>
+              Inverse of the input resolution fraction to be encoded.
+              Example: 1.0 = full resolution, 2.0 = one half of the
+              full resolution.  If unset, defaults to 1.0.
+            </p>
+          </dd>
+
+          <dt>double framerateScale<dt>
+          <dd>
+            <p>
+              Inverse of the input framerate fraction to be
+              encoded. Example: 1.0 = full framerate, 2.0 = one half
+              of the full framerate.  If unset, defaults to 1.0.
+            </p>
+          </dd>
+
+          <dt>double framerateBias</dt>
+          <dd>
+            <p>
+              What to give more bits to, if available. 0.0 = strongly
+              favor resolution or 1.0 = strongly favor framerate. 0.5
+              = neither (default).  If unset, defaults to 0.5.
+            </p>
+          </dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCRtxParameters">
+          <dt>unsigned long ssrc</dt>
+          <dd>
+            <p>
+              The SSRC of the RTP stream for RTX.  Effectively
+              readonly.  May not be changed by the application.
+            </p>
+          </dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCFecParameters">
+          <dt>unsigned long ssrc</dt>
+          <dd>
+            <p>
+              The SSRC of the RTP stream for FEC.  Effectively
+              readonly.  May not be changed by the application.
+            </p>
+          </dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCRtcpParameters">
+          <dt>DOMString cname</dt>
+          <dd>
+            <p>The Canonical Name (CNAME) used by RTCP (e.g. in SDES
+              messages).  Effectively readonly.  May not be changed by
+              the application.<p/>
+          </dd>
+
+          <dt>boolean reducedSize</dt>
+          <dd>
+            <p>Whether reduced size RTCP [RFC5506] is configured (if
+            true) or compound RTCP as specified in [RFC3550] (if
+            false).  Effectively readonly.  May not be changed by the
+            application.</p>
+          </dd>
+        </dl>
+
+        <dl class="idl" title="dictionary RTCRtpHeaderExtensionParameters">
+          <dt>DOMString uri</dt>
+          <dd>
+            <p>
+              The URI of the RTP header extension, as defined in
+              [RFC5285].  Effectively readonly.  May not be changed by
+              the application.
+            </p>
+          </dd>
+
+          <dt>unsigned short id</dt>
+          <dd>
+            <p>
+              The value put in the RTP packet to identify the header
+              extension.  Effectively readonly.  May not be changed by
+              the application.
+            </p>
+          </dd>
+
+          <dt>boolean encrypt</dt>
+          <dd>
+            <p>
+              Whether the header extension is encryted or not.
```

Comment:
I changed it to "encrypted", but we should be awkward later when we want to make it writable (as it is in ORTC, where it's named "encrypt".